Rubberduck.Parsing/Annotations/ModuleDescriptionAnnotation.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.VBEditor;
+
+namespace Rubberduck.Parsing.Annotations
+{
+    /// <summary>
+    /// Used for specifying a module's <c>VB_Description</c> attribute.
+    /// </summary>
+    public sealed class ModuleDescriptionAnnotation : AttributeAnnotationBase
+    {
+        public ModuleDescriptionAnnotation(QualifiedSelection qualifiedSelection, VBAParser.AnnotationContext context, IEnumerable<string> attributeValues)
+            : base(AnnotationType.ModuleDescription, qualifiedSelection, context, attributeValues?.Take(1).ToList())
+        {
+            Description = AttributeValues?.FirstOrDefault();
+            if ((Description?.StartsWith("\"") ?? false) && Description.EndsWith("\""))
+            {
+                // strip surrounding double quotes
```

Comment:
Should we have a string extension method for this? Seems something that might need to be done in multiple places.

Reply:
Might be a good idea.
BTW, this is basically a copy of the `DescriptionAnnotation`.